Glossary.md

```diff
@@ -0,0 +1,74 @@
+# Glossary of Terms
+
+## Domain
+
+A specific sphere of activity or knowledge. For example, cyber security, software, bills of material, licensing, software security, etc
+
+## Ontology
+
+An explicit specification of concepts, terms, categories, relationships, etc. for a particular scoped domain forming a language or basis for communication and analysis of information within the scoped domain. SPDX 3.0 is an ontology targeted to support the scoped domain of bills of material.
+
+## Class
+
+A represention of a scope/set of individual instances of a particular “concept” (e.g., File, Person, ExternalReference, etc.). Each individual instance of a class has an IRI ID and is asserted as a member of a particular class via a type statement.
+
+## Domain Class
+
+A class representing a concept of primary focus within a given domain. In a graph or serialized set of instance content this would be the granularity of what would be a node in the graph or a top-level object in the serialization set. This is typically similar to the scoping seen in labeled property graphs. In SPDX 3.0, all domain classes are subclasses of Element.
```

Comment:
> In SPDX 3.0, all domain classes are subclasses of Element.

In the licensing profile, License will be a domain class at as currently proposed will not be a subclass of Element.  Suggest removing this sentence.

Reply:
@goneall  I completely agree (and had made the remark before).

But maybe we can say that in the Core profile this holds?

Reply:
@zvr I guess so - my preference is to remove the statement, but it would not be inconsistent to qualify the statement `In the SPDX 3.0 core profile, all domain classes are subclasses of Element.`